bpo-29474: Improve documentation for weakref.WeakValueDictionary

Mariatta opened the pull request:
There were some grammatical errors in weakref.WeakValueDictionary
documentation.

This should be applied to 3.5, 3.6, and master.

Original patch was in bpo http://bugs.python.org/file46597/issue29474py3-2.patch

eliben commented:
This is a read-only mirror of CPYthon. Please see the devguide [https://docs.python.org/devguide/index.html] about contributing

Mariatta committed:
bpo-29474: Improve documentation for weakref.WeakValueDictionary

There were some grammatical errors in weakref.WeakValueDictionary
documentation.

Mariatta commented:
Not sure if I'm doing this right.
Is the idea for me to apply the labels `backport to 3.5`, and `backport to 3.6` ?

zware commented:
@Mariatta Yes, as far as I know :)
